mobilestyle.css

```diff
@@ -629,39 +639,72 @@ button:active {
   font-weight: 600;
   font-size: 150%;
   line-height: 5%;
+  flex: 0.5;
 }
 
 .skills-lis,
 .framworks-lis,
 .languages-lis {
-  flex: 0.5;
-  gap: 2%;
-  margin: 4%;
-  width: 90%;
-  display: flex;
-  flex-direction: row;
-  justify-content: space-evenly;
-  align-items: center;
-  flex-wrap: wrap;
+  flex: 1.5;
+  gap: 8px;
+  display: grid;
   padding: 2%;
-  align-content: inherit;
+  grid-template-columns: auto auto auto;
+  grid-template-rows: auto auto;
+  width: 95%;
+}
+
+.languages-lis {
+  grid-template-columns: auto auto auto auto;
+  grid-template-rows: auto;
+}
+
+.framworks-lis {
+  grid-template-columns: auto auto auto;
+  grid-template-rows: auto;
 }
 
 .framworks-lis li,
 .languages-lis li,
 .skills-lis li {
   font-family: "Inter", sans-serif;
   font-weight: 550;
-  font-size: 80%;
+  font-size: 95%;
   color: #36b37e;
   letter-spacing: 0.01em;
   background: #fff;
   border: 1px solid transparent;
-  border-radius: 5px;
-  padding: 2%;
+  border-radius: 8px;
+  padding: 5%;
   margin: 1.5%;
   white-space: nowrap;
-  flex: 1;
+  width: auto;
+}
+
+/* use this way
+.wrapper {
+  display: grid;
+  grid-template-columns: repeat(8, 1fr);
+  gap: 10px;
+  grid-auto-rows: 100px;
+  grid-template-areas:
+    "a a a a b b b b"
+    "a a a a b b b b"
+    "c c c c d d d d"
+    "c c c c d d d d";
+} */
```

Comment:
Kindly ensure to take away every commented piece of code as they make your codebase look a bit messy😉.